### DISCUSSION TITLE
Inter vs InterDisplay

### DISCUSSION THREAD

**Discussion by joergklein** [rsms/inter#776] [General]
I have programmed a website with Bootstrap and Hugo. I downloaded Inter 4.1 and I chose `InterDisplay` as the font. My website is very text-heavy. What is the advantage of InterDislpay over Inter? 

The Inter font family can also be downloaded from the Google website. Unfortunately, I could not find the current version that Google offers. `inter-v18-cyrillic_cyrillic-ext_greek_greek-ext_latin_latin-ext_vietnamese-100.woff2`.

**Comment by inferno986return** [comment]
To my understanding, the original Inter was designed for screens (effectively as an open-source replacement for Apple's San Francisco) at small sizes around 12pt (or smaller). Inter Display is meant to be for larger sizes such as headings, posters or even billboards.
